package.json

```diff
@@ -32,7 +32,8 @@
     "test:node": "npm run compile-validators && tsc && c8 mocha --es-module-specifier-resolution=node \"dist/esm/tests/**/*.spec.js\" && npm run make-coverage-badges",
     "test:node:ci": "npm run compile-validators && tsc && c8 mocha --es-module-specifier-resolution=node \"dist/esm/tests/**/*.spec.js\"",
     "test:browser": "npm run compile-validators && cross-env karma start karma.conf.cjs",
-    "test:browser-debug": "npm run compile-validators && cross-env karma start karma.conf.debug.cjs"
+    "test:browser-debug": "npm run compile-validators && cross-env karma start karma.conf.debug.cjs",
+    "license:check": "license-report --only=prod > license-report.json && node ./build/license-check.cjs"
```

Comment:
Convention: use `license-check` instead, the `:` is used when there are variants of the same task